dhis-2/dhis-api/src/main/java/org/hisp/dhis/dataanalysis/FollowupValue.java

```diff
@@ -105,4 +109,11 @@
     @JsonProperty
     private Integer max;
 
+    @JsonProperty
+    public String getPe()
+    {
+        return peType == null
+            ? null
+            : PeriodType.getPeriodTypeByName( peType ).getIsoDate( fromJdkDate( peStartDate ) );
```

Comment:
We have a centralized util method `PeriodType#getIsoPeriod` to use here.

Reply:
Changed. Also removed the `throws SQLException` from that method as discussed.